cloud-provider-openstack.spec

```diff
@@ -74,35 +74,28 @@ Summary: Provisioner for OpenStack Manila
 Provisions volumes using OpenStack Manila API.
 
 %prep
+# cloud-provider-openstack assumes it's compiled in a Go workspace
+# Let's create one
 %if 0%{do_prep}
-%setup -q
+%setup -n cloud-provider-openstack
+mkdir -p src/k8s.io
+for dep in $(ls vendor); do
```

Comment:
This is awfully complicated. All our other packages use this to create GOPATH:

```
mkdir -p src/%{provider}.%{provider_tld}/%{project}
ln -s ../../../ src/%{import_path}
export GOPATH=$(pwd):$(pwd)/repo-infra/vendor:%{gopath}
```

http://pkgs.devel.redhat.com/cgit/rpms/csi-attacher/tree/csi-attacher.spec?h=rhaos-3.10-rhel-7#n53


You should not unroll vendor/ into src/.

Reply:
This whole think is actually a side-effect of the testing builds: "make manila-provisioner" expect to be run from a "proper" go workspace so i created one so i could fiddle with things after %prep...

Will simplify it. Thanks.